torch/_inductor/ir.py

```diff
@@ -3072,6 +3072,8 @@ def create(cls, kernel, *args, **kwargs):
         assert tensor_args or isinstance(
             example_output, torch.Tensor
         ), "Not sure where to find device info"
+        req_stride_order = get_stride_order(example_output.stride())
+        tensor_args[:] = [cls.require_stride_order(t, req_stride_order) for t in tensor_args]
```

Comment:
We'd better check why `example_output` has contiguous stride instead of channels last contiguous stride in this case.